src/runtime/rpc/rpc_server_env.cc

```diff
@@ -4,11 +4,154 @@
  * \brief Server environment of the RPC.
  */
 #include <tvm/runtime/registry.h>
```

Comment:
The filesys related code is not really cross-platform and can be moved to the src/apps/cpp_rpc